package com.zelenux.upprpo_server_test.viewer.exceptions.userExceptions;

public class UserAlreadyExistsException extends UserException {
}
